src/__init__.py

```diff
@@ -1,8 +1,24 @@
 from pkg_resources import get_distribution
+import codecs
+import json
 
 __version__ = get_distribution('rasa_nlu').version
 
 
 class Interpreter(object):
     def parse(self, text):
         raise NotImplementedError()
+
+    @staticmethod
+    def load_synonyms(entity_synonyms):
+        if entity_synonyms:
+            with codecs.open(entity_synonyms, encoding='utf-8') as infile:
+                return json.loads(infile.read())
+
+    @staticmethod
+    def replace_synonyms(entities, entity_synonyms):
+        for i in range(len(entities)):
+            entity_value = entities[i]["value"]
+            if (type(entity_value) == unicode and type(entity_synonyms) == unicode and
+                    entity_value.lower() in entity_synonyms):
+                entities[i]["value"] = entity_synonyms[entity_value]
```

Comment:
I do understand the if, but the access into the dict should be `entity_synonyms[entity_value.lower()]` otherwise it might fail
(`entity_value.lower() in entity_synonyms` might be true in your if stmt, but `entity_value in entity_synonyms` can be false at the same time).

Concerning entities with numbers as value: 
- is there a reason why we need to support non string values for entities?  @amn41
- if we do support it, do we have an example for that in the test data? there should definitely be a test for that (esp. with this synonym logic)